changes/428.feature

```diff
@@ -0,0 +1 @@
+- Modify the loading process so that the scheduler can be loaded reflecting scheduler_opts.
```

Comment:
This news fragment string is merged into a Markdwon-formatted list when released, via [towncrier](https://pypi.org/project/towncrier/). So you don't have to put the starting `- ` because `* ` will be added automatically. :wink: